### DISCUSSION TITLE
verusdoc: hiding internal functions for traits

### DISCUSSION THREAD

**Discussion by jaybosamiya** [verus-lang/verus#607] [General]
We probably want to hide the `VERUS_SPEC__*` functions from the documentation (`#[doc(hidden)]` or similar). 

For example, see https://verus-lang.github.io/verus/verusdoc/vstd/slice/trait.SliceAdditionalSpecFns.html which currently shows:

<img width="657" alt="image" src="https://github.com/verus-lang/verus/assets/5683582/2867b868-1ae3-4dce-a3c1-afdb4ef1f0b0">

**Comment by tjhance** [comment]
should be a pretty easy fix - is there much to discuss?

**Reply by jaybosamiya** [reply]
I don't think so, that's why I'd opened it as an issue https://github.com/verus-lang/verus/issues/602

I thought it was a simple fix and since you're familiar with the verusdoc stuff, I assigned you to it since I thought it'd be an easy fix for you

**Comment by tjhance** [comment]
resolved in a67e659e96cb126ce41377b581644e2bd78b0cc4
